futurex_openedx_extensions/dashboard/views.py

```diff
@@ -445,6 +445,14 @@ def get(self, request: Any, *args: Any, **kwargs: Any) -> JsonResponse:  # pylin
         return JsonResponse(get_tenants_info(tenant_ids))
 
 
+class AccessibleTenantsInfoViewV2(FXViewRoleInfoMixin, AccessibleTenantsInfoView):
+    """View to get the list of accessible tenants version 2"""
+    permission_classes = [FXHasTenantCourseAccess]
+    fx_view_name = 'accessible_info'
+    fx_default_read_only_roles = ['staff', 'instructor', 'data_researcher', 'org_course_creator_group']
```

Comment:
System admins will be allowed by default through `FXHasTenantCourseAccess`; all others should be blocked
```suggestion
    fx_default_read_only_roles = []
```